### ISSUE TITLE
Typo in neural_style loss function (style_score + style_score)

### ISSUE THREAD

**Issue by josephreisinger** [pytorch/tutorials#117]
Based on my understanding of the optimization,
```
return style_score + style_score
```
should read:
```
return style_score + content_score
```
on line: https://github.com/pytorch/tutorials/blob/master/advanced_source/neural_style_tutorial.py#L584

**Comment by chsasank** [comment]
indeed, thanks! Will fix that.
